rfd/0084-oracle-db-access-integration.md

```diff
@@ -0,0 +1,107 @@
+---
+authors: Marek Smoliński (marek@goteleport.com)
+state: draft
+---
+
+# RFD 84 - Teleport Oracle Access Integration
+
+## What
+
+
+This RFD proposes a way to integrate Oracle Database Access with Teleport.
+Oracle DB Access differs from currently supported database protocols by Teleport Database Access.
+Namely, Oracle DB protocol is private and there is not any official documentation describing the Oracle protocol.
+
+## Why
+
+We want to increase the number of databases supported by Teleport and allow users to connect to Oracle using Teleport Database Access.
+Where we want to provide a way for users to connect to Oracle databases through Teleport using the same Teleport UI/UX as for other supported databases.
```

Comment:
I would rephrase this a little to highlight mutual benefits e.g. something like:

```
Adding Oracle support to Teleport Database Access would allow our shared customer bases to take advantage of Teleport's access controls and audit logging capabilities when accessing their Oracle databases. 

* Administrators will be able to control access to their entire fleet of database servers through their identity provider with SSO.
* Users will be able to connect to Oracle databases with the tools they're familiar with without having to deal with passwords or shared secrets.
* Auditors will be able to view the database activity and tie it to a particular user identity within an organization.
```